src/qt/pivx/send.cpp

```diff
@@ -249,22 +249,25 @@ void SendWidget::clearEntries(){
     addEntry();
 }
 
-void SendWidget::addEntry(){
-    if(entries.isEmpty()){
-        createEntry();
+void SendWidget::addEntry() {
+    if (entries.isEmpty()) {
+        SendMultiRow *sendMultiRow = createEntry();
+        sendMultiRow->setFocus();
     } else {
         if (entries.length() == 1) {
             SendMultiRow *entry = entries.at(0);
             entry->hideLabels();
             entry->setNumber(1);
-        }else if(entries.length() == MAX_SEND_POPUP_ENTRIES){
+            entry->setFocus();
```

Comment:
No need to setFocus, this entry is the first one and the code is creating a new one in this method.